RELEASE.md

```diff
@@ -101,14 +101,12 @@ For a **PATCH** release:
 - Create a tag using the `HEAD` of the `release-x.x.x` branch. This can be done using the `git` CLI or
   Github's [release][release] page.
 - The Release will be automatically created, after that is complete do the following:
-  - run `make build-crd-install-yaml` and attach the crd install yaml for the version to the release.
-  - manually build and attach a `bpfd-operator-install-vx.x.x.yaml` to provide an easy to use install
-    yaml for the bpfd-operator.
-  - manually build and attach the following files:
-    - `go-xdp-counter-install-vx.x.x.yaml`
-    - `go-tc-counter-install-vx.x.x.yaml`
-    - `go-tracepoint-counter-install-vx.x.x.yaml`
-    so that the examples for a given release can be easily deployed.
+  - run `make build-release-yamls` and attach the yamls for the version to the release. These will include:
+    - `bpfd-crds-install-v0.3.0.yaml`
+    - `bpfd-operator-install-v0.3.0.yaml`
+    - `go-xdp-counter-install-v0.3.0.yaml`
+    - `go-tc-counter-install-v0.3.0.yaml`
+    - `go-tracepoint-counter-install-v0.3.0.yaml`
```

Comment:
I think these should all be x.x.x like they were before.

Reply:
Done!